packages/ui/react-ui-table/src/util/dynamic-table.ts

```diff
@@ -36,13 +39,13 @@ export const getBaseSchems = ({
   typename?: string;
   properties?: TablePropertyDefinition[];
   jsonSchema?: JsonSchemaType;
-  schema?: BaseSchema;
+  schema?: Schema.Schema.AnyNoContext;
 }): { typename: string; jsonSchema: JsonSchemaType } => {
   if (typename && properties) {
     const schema = getSchemaFromPropertyDefinitions(typename, properties);
     return { typename: schema.typename, jsonSchema: schema.jsonSchema };
   } else if (schema) {
-    return { typename: schema.typename, jsonSchema: schema.jsonSchema };
+    return { typename: getTypename(schema)!, jsonSchema: toJsonSchema(schema) };
```

Comment:
Prefer not to use non-null assertion operator.

Reply:
see above